### PR TITLE
#1185 Initialize P0 in Ac emulation

### PR DESCRIPTION
**Pull request by MarianneSaugierRTE**

### CODE REVIEW

**Review by cossartque — COMMENTED** [review]
Ok for me.
I think you should also remove PRefSetPu in the .par file of the test case as it is not used anymore.

File: examples/DynaFlow/IllustrativeExamples/HVDC_CLA/reference/outputs/curves/curves.csv [hunk 1]
```diff
@@ -1,34 +1,34 @@
 time;NETWORK__BUS____7-BUS___10-2_AC_iSide2;HVDC_hvdc_P1Pu;
-0.000000;1377.507418;11.955000;
```
Comment (cossartque) [inline examples/DynaFlow/IllustrativeExamples/HVDC_CLA/reference/outputs/curves/curves.csv:2]: Is it the only nrt impacted ?
What about the HVDC_PhaseShifter ?

**Comment by MarianneSaugierRTE** [comment]
> Ok for me.
> I think you should also remove PRefSetPu in the .par file of the test case as it is not used anymore.

ok!

File: examples/DynaFlow/IllustrativeExamples/HVDC_CLA/reference/outputs/curves/curves.csv (hunk 1 above)
Comment (MarianneSaugierRTE) [reply examples/DynaFlow/IllustrativeExamples/HVDC_CLA/reference/outputs/curves/curves.csv:2]: HVDC_PhaseShifter was not impacted according to the nrt report, I will double check

Comment (cossartque) [reply examples/DynaFlow/IllustrativeExamples/HVDC_CLA/reference/outputs/curves/curves.csv:2]: Ok thanks!

Comment (MarianneSaugierRTE) [reply examples/DynaFlow/IllustrativeExamples/HVDC_CLA/reference/outputs/curves/curves.csv:2]: I confirm ;)

**Comment by MarianneSaugierRTE** [comment]
> > Ok for me.
> > I think you should also remove PRefSetPu in the .par file of the test case as it is not used anymore.
> 
> ok!

done!

**Review by cossartque — APPROVED** [review]

**Review by adriguir — APPROVED** [review]

**Review by adriguir — APPROVED** [review]

**Pull request merged**